modules/apps/portal/portal-upload/src/main/java/com/liferay/portal/upload/internal/ServletFileUploadImpl.java

```diff
@@ -63,6 +70,14 @@ public List<FileItem> parseRequest(
 
 			return fileItems;
 		}
+		catch (FileUploadBase.SizeLimitExceededException
```

Comment:
By catching this exception here we are not running the `if` in the catch below, therefore `uploadException.setExceededUploadRequestSizeLimit(true);` is not being run anymore.